### PR TITLE
chore: add transfer agreement phase calls

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/interface/thirdparty-pisp-api-template.yaml [hunk 1]
```diff
@@ -127,3 +127,215 @@ paths:
...
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
+        503:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
+  /authorizations:
+    post:
+      operationId: AuthorizationPostRequest
+      summary: AuthorizationPostRequest
+      description: |
+        Switch -> PISP
+      parameters:
```
Comment (lewisdaly) [inline src/interface/thirdparty-pisp-api-template.yaml:214]: Are there meant to be more parameters here?

File: src/interface/thirdparty-pisp-api-template.yaml [hunk 2]
```diff
@@ -127,3 +127,215 @@ paths:
...
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/404'
+        405:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/405'
+        406:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/406'
+        501:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
+        503:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
+  /thirdpartyRequests/transactions/{ID}/authorizations:
```
Comment (lewisdaly) [inline src/interface/thirdparty-pisp-api-template.yaml:243]: This isn't a part of the `thirdparty-pisp-api`. It's called by the DFSP

File: src/interface/thirdparty-pisp-api-template.yaml [hunk 3]
```diff
@@ -127,3 +127,215 @@ paths:
...
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/404'
+        405:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/405'
+        406:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/406'
+        501:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/501'
+        503:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/503'
+    put:
```
Comment (lewisdaly) [inline src/interface/thirdparty-pisp-api-template.yaml:294]: This isn't a part of the `thirdparty-pisp-api`. It's called by the DFSP

**Review by lewisdaly — CHANGES_REQUESTED** [review]

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 2 above)
Comment (kleyow) [reply src/interface/thirdparty-pisp-api-template.yaml:243]: Yes but doesn't the PISP also receive POST `/thirdpartyRequests/transactions/{ID}/authorizations` from the switch?
pisp-PISPTransferDetailedAPI-page2-png-at-master-·-mojaloop-pisp

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 3 above)
Comment (kleyow) [reply src/interface/thirdparty-pisp-api-template.yaml:294]: The PISP responds to the Switches' post request with a PUT. Or was this changed?
pisp-PISPTransferDetailedAPI-page2-png-at-master-·-mojaloop-pisp (1)

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 1 above)
Comment (kleyow) [reply src/interface/thirdparty-pisp-api-template.yaml:214]: Honestly. I've been meaning to ask that I'm unsure what headers belong on what requests.
For `/authorizations` I've added the ones that I missed.

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 2 above)
Comment (lewisdaly) [reply src/interface/thirdparty-pisp-api-template.yaml:250]: I think that's the auth service.
Screen Shot 2020-09-18 at 9 42 56 am

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 3 above)
Comment (lewisdaly) [reply src/interface/thirdparty-pisp-api-template.yaml:301]: same as above - that's the auth service

**Comment by kleyow** [comment]
I actually think I may be blind...thanks for clarifying.

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 3 above)
Comment (kleyow) [reply src/interface/thirdparty-pisp-api-template.yaml]: removed.

File: src/interface/thirdparty-pisp-api-template.yaml (hunk 2 above)
Comment (kleyow) [reply src/interface/thirdparty-pisp-api-template.yaml]: removed.

**Comment by lewisdaly** [comment]
> I actually think I may be blind...thanks for clarifying.

Haha or they're just super squishy sequence diagrams

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**